.github/ISSUE_TEMPLATE/resolver-failure.md

```diff
@@ -5,16 +5,30 @@ labels: ["K: UX", "K: crash", "C: new resolver", "C: dependency resolution"]
 ---
 
 <!--
-  Please provide as much information as you can about your failure, so that we can understand the root cause.
+Please provide as much information as you can about your failure, so that we can understand the root cause.
 
-  For example, if you are installing packages from pypi.org, we'd like to see:
+Try if your issue has been fixed in the in-development version of pip. Use the following command to install pip from master:
 
-  - Your terminal output
-  - Any inputs to pip, for example:
-    - any package requirements: any CLI arguments and/or your requirements.txt file
-    - any already installed packages, outputted via `pip freeze`
+    python -m pip install -U "pip @ git+https://github.com/pypa/pip.git"
```

Comment:
```suggestion
    python -m pip install -U "pip @ https://github.com/pypa/pip/archive/master.zip"
```
It doesn't require to have git available and to clone the whole project history.